crates/jstzd/src/task/octez_client.rs

```diff
@@ -97,6 +101,61 @@ impl fmt::Display for Signature {
     }
 }
 
+#[derive(Debug)]
+pub struct Address {
+    pub hash: public_key_hash::PublicKeyHash,
+    pub public_key: public_key::PublicKey,
+    pub secret_key: Option<secret_key::SecretKey>,
+}
+
+impl Address {
+    const HASH: &'static str = "Hash";
+    const PUBLIC_KEY: &'static str = "Public Key";
+    const SECRET_KEY: &'static str = "Secret Key";
+}
+
+impl TryFrom<StdOut> for Address {
+    type Error = anyhow::Error;
+    // the output of `show address` command is expected to be in the following format:
+    /*
+     * Hash: tz1..
+     * Public Key: edpk..
+     * Secret Key: unencrypted:edsk..
```

Comment:
Maybe change this to
```
* Secret Key: (unencrypted:)edsk..
```
since secret keys are not always unencrypted?

Reply:
nice